Jenkinsfile

```diff
@@ -1 +1,5 @@
-buildPlugin(useAci: false, configurations: buildPlugin.recommendedConfigurations())
+buildPlugin(useAci: false, configurations: [
+  [ platform: "linux", jdk: "8", jenkins: null ],
+  [ platform: "windows", jdk: "8", jenkins: null ],
+  [ platform: "linux", jdk: "11", jenkins: "2.222.3", javaLevel: 8 ]
```

Comment:
There might be some context I've missed in the other PRs that led up to this one, but I'll ask anyway. Why specify a Jenkins version here, but not in the other two?

Reply:
`jenkins: null` means to use the version specified in `pom.xml` under `<jenkins.version>`. We could explicitly specify `2.176.4` here instead, but then anytime we updated the version we would need to update both `pom.xml` and `Jenkinsfile`, which is easy to forget (not really a problem since the PR build will fail until you fix it, it's just a bit tedious). This way, we only need to update `Jenkinsfile` if we want to test on a newer version than `2.222.3`.

Reply:
Oh and the big idea here is to test on Windows and Linux using Java 8 and the minimum required Jenkins version, then on Linux using Java 11 and a much newer LTS to make sure we don't break things for the new LTS or for Java 11.